documentation/src/main/asciidoc/reference/mapper-orm-indexing-jsr352.asciidoc

```diff
@@ -3,4 +3,473 @@
 // Search 5 anchors backward compatibility
 [[jsr352-integration]]
 
```

Comment:
I see what you tried to do here; you wanted this file to be considered as "renamed". It didn't work because the file existed before this commit.
Just insert a commit that removes this file, before this commit. In this commit, rename `manual-index.asciidoc` to `mapper-orm-indexing-jsr352.asciidoc`, and change the anchors on the main heading (as above). Then Git should detect the renaming just fine.